boot/zephyr/CMakeLists.txt

```diff
@@ -95,8 +95,8 @@ zephyr_library_sources(
 
 if(NOT DEFINED CONFIG_FLASH_PAGE_LAYOUT)
   zephyr_library_sources(
-	flash_map_legacy.c
-	)
+    flash_map_legacy.c
```

Comment:
Please pull out the tab/space patch into its own patch.

Reply:
Done.